javascript-algorithms-and-data-structures/learn-basic-javascript-by-building-a-pyramid-generator/script.js

```diff
@@ -0,0 +1,123 @@
+// teach these different variable types.
+const symbol = "#"
+const count = 8;
+const rows = [];
+// Remove this after #1 is done.
+let done = 0;
+// Add this only during #3
+// note that this should be a const, but our tests need it to be a let :3
+let inverted = true;
+
+// This function will be built during the first loop.
+// Explain scoping of `count` variable here.
+// Then explain "variable shadowing"? Might be too complex. Will consider.
```

Comment:
yeah, that might be a little much for this first project.
We can always teach it in variable shadowing in the RPG project 👍

Reply:
Cool, I can skip that explanation :3